src/core/Modal/ModalContent/ModalContent.baseStyles.tsx

```diff
@@ -1,7 +1,10 @@
 import { css } from 'styled-components';
 import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
 
 export const baseStyles = (theme: SuomifiTheme) => css`
+  ${element(theme)}
+  ${font(theme)('bodyText')}
```

Comment:
Should the body text size take into account the small screen option as well, since the component offers that in the API?

Reply:
Apparently the current design guideline is that all body text is 18px regardless of viewport size although the documentation says otherwise. Let's go with the current guidelines and keep it 18px here as well :)